### PR TITLE
add nix-du

### PR DESCRIPTION
**Pull request by asymmetric**
https://github.com/symphorien/nix-du

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,7 @@
 * [nixpkgs-hammering](https://github.com/jtojnar/nixpkgs-hammering) - An opinionated linter for Nixpkgs package expressions.
 * [nix-alien](https://github.com/thiagokokada/nix-alien) - Run unpatched binaries on Nix/NixOS easily.
 * [nix-diff](https://github.com/Gabriella439/nix-diff) - A tool to explain why two Nix derivations differ.
+* [nix-du](https://github.com/symphorien/nix-du) - Visualise which gc-roots to delete to free some space in your nix store.
```
Comment (houstdav000) [inline README.md:95]: Capitalize "nix", and it's good.

**Review by houstdav000 — CHANGES_REQUESTED** [review]
Just needs "nix" capitalized.

File: README.md (hunk 1 above)
Comment (asymmetric) [reply README.md:95]: I took it verbatim from the project description though. Do you still want me to capitalize?

**Comment by houstdav000** [comment]
Yeah, capitalize it still

**Review by houstdav000 — APPROVED** [review]
Looks good now.

**Pull request merged**